MediaBrowser.Model/Notifications/NotificationOption.cs

```diff
@@ -15,6 +15,14 @@ public NotificationOption(string type)
             SendToUsers = Array.Empty<string>();
         }
 
+        public NotificationOption()
+        {
+            Type = string.Empty;
```

Comment:
```suggestion
            Type = null;
```

Reply:
Shouldn't we just remove Type as it's null anyway?